cloudflare.py

```diff
@@ -430,15 +454,15 @@ def append_in_order(ops, order):
             for op in sorted(ops, key=lambda op: order[op["action"]]):
                 result.append(op)
 
-        for name, ops in groups["primary"].iteritems():
```

Comment:
Hey @cloudflare, please merge this PR, the current master does not work and need this change.